joint_limits_interface/package.xml

```diff
@@ -2,22 +2,26 @@
   <name>joint_limits_interface</name>
   <version>0.15.1</version>
   <description>Interface for enforcing joint limits.</description>
+
   <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
   <maintainer email="enrique.fernandez.perdomo@gmail.com">Enrique Fernandez</maintainer>
   <maintainer email="mathias.luedtke@ipa.fraunhofer.de">Mathias Lüdtke</maintainer>
+
   <license>BSD</license>
 
   <url type="website">https://github.com/ros-controls/ros_control/wiki</url>
   <url type="bugtracker">https://github.com/ros-controls/ros_control/issues</url>
   <url type="repository">https://github.com/ros-controls/ros_control</url>
+
   <author>Adolfo Rodriguez Tsouroukdissian</author>
 
+
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>roscpp</depend>
   <depend>hardware_interface</depend>
-  <depend>urdf</depend>
   <depend>liburdfdom-dev</depend>
```

Comment:
Could you please have a look at which packages can be reduced to a build_depend? I think everything with a `-dev` is a good candidate

Reply:
`liburdfdom-dev` is a dependency of `urdf`, and our use of it explicitly is a leftover from [a 2014 packaging issue](https://github.com/ros/rosdistro/issues/4633). That issue is long resolved, and Indigo is EOL, so I think we're good to drop it (7ca25abf).

I will go through the other dependencies and see what can be reduced though.

Reply:
Went through and update all the dependencies in b3654b3b

Reply:
Thank you!